File: src/main/java/com/twoguis/carfixer/model/Usuario.java
package com.twoguis.carfixer.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Usuario {
    private int id_usuario;
    private String nome;
    private String cpf;
    private String email;
    private String telefone;
    private String senha;
    private boolean permission;
}

File: src/app/model/usuario.ts
export class Usuario {
    id: number;
    nome: string;
    cpf: string;
    telefone: string;
    email: string;
    senha: string;
    permission: number; //create a Enum class

    constructor() {
        this.id = 0;
        this.nome = "";
        this.cpf = "";
        this.email = "";
        this.telefone = "";
        this.senha = "";
        this.permission = 0;
    }
}
